services/compute/compute.go

```diff
@@ -769,6 +769,10 @@ type GalleryImageProperties struct {
 	ContainerName *string `json:"containername,omitempty"`
 	//Type of source of gal image (sfs/http/local)
 	SourceType common.ImageSource `json:"sourceType,omitempty"`
+	// CloudInitDataSource - READ-ONLY; The cloud init data source to be used with the image. [NoCloud, Azure]. Default Value – NoCloud. For marketplace images it will be Azure.
+	CloudInitDataSource common.CloudInitDataSource `json:"cloudInitDataSource,omitempty"`
+	// DownlaodStatus - Current download status of the image
+	DownloadStatus common.DownloadStatus `json:"downloadStatus,omitempty"`
```

Comment:
Try to add DownloadStatus into Status key value map
It doesnt make sense here

Reply:
Should this be DownloadStatus or upload status - since we are uploading the image to MOC

Reply:
It's tricky. In doing that it's also downloading the image from Azure Marketplace. In the ARM API it's proposed as DownloadStatus.

Reply:
I'll move downloadstatus under status but will need to modify moc/pkg/status to pass it down. I'll link the pr here once ready.

Reply:
@pathuang-sia  - I think the ask was different and Manoj answerd it. Thanks.

Reply:
https://github.com/microsoft/moc/pull/116